src/components/serverPaginatedTable/serverPaginatedTable.js

```diff
@@ -169,26 +204,50 @@ class ServerPaginatedTableView extends React.Component {
   async fetchData(offset, rowsRequired, sortOrder = {}) {
     let sortDirection = 'asc';
     let sortColumn = 'arm';
-
+    let offsetReal = offset;
+    let page = offset / rowsRequired;
+    // if the offset value is bigger that the count, then change offset value
+    if (offset >= this.props.count) {
+      if (this.props.count % rowsRequired !== 0) {
```

Comment:
page = this.props.count % rowsRequired !== 0 ? Math.floor(this.props.count / rowsRequired) :Math.floor(this.props.count / rowsRequired) - 1;
much cleaner way

Reply:
let me try